db/error_handler_fs_test.cc

```diff
@@ -851,13 +851,17 @@ TEST_F(DBErrorHandlingFSTest, DoubleManifestWriteError) {
       });
   SyncPoint::GetInstance()->EnableProcessing();
   s = Flush();
-  ASSERT_EQ(s.severity(), ROCKSDB_NAMESPACE::Status::Severity::kHardError);
+  ASSERT_TRUE(s.IsNoSpace());
+  ASSERT_EQ(dbfull()->TEST_GetBGError().severity(),
```

Comment:
For error severity, the only documentation that I found is "If the database instance goes into read-only mode, the following foreground operations will return the error status on all subsequent calls -" from https://github.com/facebook/rocksdb/wiki/Background-Error-Handling#detection which suggests errors returned after DB turns into read-only mode should have the severity set. In that case, I believe we return bg_error_ from error_handler which will have the severity.